tools/docker/ubuntu18.04-ci-gpu.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gluonai/gluon-nlp:gpu-base-latest
+
+LABEL maintainer="GluonNLP Team"
+
+WORKDIR ${WORKDIR}/gluon-nlp
```

Comment:
Since `${WORKDIR}` is added in the work path, we need to update some commands by adding `/workspace` before `/gluon-nlp` in some shell scripts([compile_notebooks.sh](https://github.com/dmlc/gluon-nlp/blob/d60dae30ecdac847c609309c284517cb5a64d520/tools/batch/batch_states/compile_notebooks.sh#L30), [test.sh](https://github.com/dmlc/gluon-nlp/blob/d60dae30ecdac847c609309c284517cb5a64d520/tools/batch/batch_states/test.sh#L9)), where absolute path is used. Otherwise, these CI tests will fail.

Reply:
I can make a follow-up pr to fix CI after this pr is merged.